bag_replicator/src/main/scala/uk/ac/wellcome/platform/archive/bagreplicator/services/BagReplicatorWorker.scala

```diff
@@ -72,10 +69,11 @@ class BagReplicatorWorker[
 
       srcPrefix = payload.bagRoot
 
-      dstPrefix = destinationBuilder.buildDestination(
-        storageSpace = payload.storageSpace,
-        externalIdentifier = payload.externalIdentifier,
-        version = payload.version
+      dstPrefix = DestinationBuilder.buildDestination(
+        namespace = destinationConfig.namespace,
+        payload.storageSpace,
+        payload.externalIdentifier,
+        payload.version
```

Comment:
nit: put the keyword arguments back please?

Reply:
Got lost as this was temporarily curried, where keyword args are not allowed.